backend/core/utils.py

```diff
@@ -467,7 +467,7 @@ def _upload_directory(self, directory_path, bucket_name):
 def get_email_message(obj_instance, status):
     hostname = settings.FRONTEND_URL
     # training_model_url = f"{hostname}/ai-models/{training_instance.model.id}"
-    profile_url = f"{hostname}/profile"  # doing profile for now as training url won't work for the predictions, i can read from the class name and filter out the condition , keeping it simple for now can do that later #todo
+    profile_url = f"{hostname}/profile/offline-predictions"  # doing profile for now as training url won't work for the predictions, i can read from the class name and filter out the condition , keeping it simple for now can do that later #todo
```

Comment:
This message is dynamic , so for both training and offline predictions, hence with this it will send out the offline predictions link for the training as well ! 😉 , here object instance type needs to be captured and have separate condition for training and prediction

Reply:
No worries its a small fix , I have added the condition !